mne/forward/forward.py

```diff
@@ -778,6 +784,13 @@ def write_forward_solution(fname, fwd, overwrite=False, verbose=None):
     See Also
     --------
     read_forward_solution
+
+    .. note:: Forward solutions with free orientation are always stored on disk
```

Comment:
This should be a proper [Notes section](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)